packages/main/src/types/IconColor.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * Different Icon semantic colors.
+*
+* @class
+* @enum {string}
+* @public
+* @author SAP SE
+* @alias sap.ui.webc.main.types.IconColor
+*/
+enum IconColor {
+	/**
+	 * Contrast color
+	 * @public
+	 * @type {Contrast}
+	 */
+	Contrast = "Contrast",
+
+	/**
+	 * Critical color
+	 * @public
+	 * @type {Critical}
+	 */
+	Critical = "Critical",
+
+	/**
+	 * Default color (brand color)
+	 * @public
+	 * @type {Default}
+	*/
+	Default = "Default",
+
+	/**
+	 * info type
+	 * @public
+	 * @type {Information}
+	 */
+	Information = "Information",
+
+	/**
+	 * Color for icon used as a marker
+	 * @public
+	 * @type {Marker}
+	 */
+	Marker = "Marker",
+
+	/**
+	 * Negative color
+	 * @public
+	 * @type {Negative}
+	 */
+	Negative = "Negative",
+
+	/**
+	 * Neutral color
+	 * @public
+	 * @type {Neutral}
+	 */
+	Neutral = "Neutral",
+
+	/**
+	 * Color that indicates an icon which isn't interactive
+	 * @public
+	 * @type {NonInteractive}
+	 */
+	NonInteractive = "NonInteractive",
+
+	/**
+	 * Positive color
+	 * @public
+	 * @type {Positive}
+	 */
+	Positive = "Positive",
+
+	/**
+	 * Color for icon used in a Tile
+	 * @public
+	 * @type {Tile}
+	 */
+	Tile = "Tile"
```

Comment:
Not sure if "Tile" type makes sense for the Icon web component as we don't have a Tile component and this seems to be linked with the Tile. Marker seems very specific too. I mean if we want to have them somehow we need to explain what are these types of color.
Positive, Negative, Critical, Neutral, Default, Information, Contrast are generic, but Marker and Tile seem odd to me. 
In SAPUI5 we have them, but perhaps for very specific usages that don't exist at the moment in our case.
Could you discuss this matter

Reply:
Hello @ilhan007,
We discussed this briefly with @plamenivanov91 and I understand but perhaps having them won't hurt to keep this types and they would rather be useful in the future. Additionally, we would be consistent with the UI5 implementation and it would be simpler to think about the APIs.
Best regards,
Jordan

Reply:
Thank you @jdichev Jordan for adding context, I see. They won't harm indeed, but if we see they are really needed, we can add always add them. Removing the is more difficult as they are public. This is why not adding them now is what I suggest, but the final decision is yours of course.